app/main/views.py

```diff
@@ -21,19 +20,58 @@ def get_map():
     pictures = []
     descriptions = []
 
+    """Create a new form."""
+    form = NewIncidentForm()
+    current_loc = "testing123"
```

Comment:
This code is pretty confusing and messy. Why are you doing imports inside the view function? What are you faking?